pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/NamespaceStatsAggregator.java

```diff
@@ -70,7 +71,19 @@ public static void generate(PulsarService pulsar, boolean includeTopicMetrics, b
         LongAdder topicsCount = new LongAdder();
```

Comment:
I think it would make sense to rename this variable as part of this work so that it is clear what the `topicsCount` is counting.

Reply:
Indeed, renaming the metrics name can not only have a clear meaning, but also separate the data of the broker from that of the topic in the prometheus, greatly improving the performance of querying the broker metrics.

Reply:
I meant that we should rename this variable. With the most recent commit, it's less ambiguous, so you can ignore this comment.